lsdcGui.py

```diff
@@ -4427,8 +4426,8 @@ def addSampleRequestCB(self,rasterDef=None,selectedSampleID=None):
 
       if (getBlConfig("queueCollect") == 0):
         if (self.mountedPin_pv.get() != self.selectedSampleID):                    
+          self.selectedSampleID = self.mountedPin_pv.get()
           self.popupServerMessage("You can only add requests to a mounted sample, for now.")
```

Comment:
Did Edwin want these popup messages to be removed? It seems like the behavior is different now, so it should either not be displayed at all or show something different.